registry.json

```diff
@@ -1842,5 +1842,34 @@
       "clientPublic": "https://rpc-osmosis.keplr.app/",
       "clientDocs": ""
     }
+  },
+  {
+    "id": "ecash",
+    "name": "ECash",
+    "coinId": 899,
+    "symbol": "XEC",
+    "decimals": 2,
+    "blockchain": "Bitcoin",
+    "derivationPath": "m/44'/899'/0'/0/0",
+    "curve": "secp256k1",
+    "publicKeyType": "secp256k1",
+    "p2pkhPrefix": 0,
+    "p2shPrefix": 5,
+    "hrp": "ecash",
+    "publicKeyHasher": "sha256ripemd",
+    "base58Hasher": "sha256d",
+    "xpub": "xpub",
+    "xprv": "xprv",
+    "explorer": {
+      "url": "https://explorer.bitcoinabc.org",
+      "txPath": "/tx/",
+      "accountPath": "/address/"
+    },
+    "info": {
+      "url": "https://e.cash",
+      "source": "https://github.com/trezor/blockbook",
+      "rpc": "",
```

Comment:
We don't have one. We have electrum servers (fulcrum). Would that work?

Reply:
we don't use electrum in the app tho

Reply:
I submitted a PR to blockbook to have them support eCash: https://github.com/trezor/blockbook/pull/731. I initially assumed that running it in BCH mode with an eCash node would work, but it turns out that there are some settings regarding the number of decimals (base unit) that would maybe cause issues.

Unfortunately there is still no review on my blockbook PR after 3 weeks.

Do you know of any alternative way that would work for you guys? It seems to me that most of the requirements you have would only need a simple public node (which we have), except maybe for an address indexer.

Reply:
try tag https://github.com/martinboehm at your pr?

Reply:
@hewigovens We're still blocked on this (although I tried pinging them yesterday).

Are there any other options other than getting the trezor/blookbook PR merged? If the eCash team set up a public (compatible) blockbook node, would that work to then have eCash integrated into Trust Wallet? Any other options?

Reply:
Yeah, that works too

Reply:
eCash support has been added to the latest blockbook master:

https://github.com/trezor/blockbook/pull/731

Reply:
Yes, could you please update wallet core registry.json? Add it to the `rpc` field of `ecash`, it will be benefit to the community

Reply:
https://github.com/trustwallet/wallet-core/pull/2130

Reply:
We'd like to have eCash supported properly in the mobile wallet. Since it's closed source, is there anyone in particular we should be talking to about that? It looks like we're all set up in wallet-core but are still not listed on mobile.

Reply:
Decisions for inclusion in the app is based on several considerations (business, etc.).
You may have a look at https://governance.trustwallet.com/#/  (community-lead proposals).